src/cmd/manage_nsfs.js

```diff
@@ -96,8 +96,6 @@ async function fetch_bucket_data(action, user_input) {
         // added undefined values to keep the order the properties when printing the data object
         _id: undefined,
         name: user_input.name === undefined ? undefined : String(user_input.name),
-        owner_account: undefined,
```

Comment:
I think we had this "owner_account" on purpose to have the property saved as undfined.